frontend/coreproject/src/routes/myanimelist/[id]/+page.ts

```diff
@@ -0,0 +1,5 @@
+export const load = ({ params }) => {
+  return {
+      id: params.id
+  }
+}
```

Comment:
What's this for?

Reply:
To get id from url dynamically

Reply:
Thats not how you do things in svelte. Look into [svelte page store](https://linguinecode.com/post/how-to-access-sveltekit-dynamic-params-from-router)

Reply:
Yea! I know that, I thought page.ts file will be useful later